performance_regression_test.py

```diff
@@ -319,6 +323,18 @@ def run_mixed_workload(self, nemesis=False):
         self.display_results(results, test_name='test_latency')
         self.check_regression()
 
+    @latency_calculator_decorator
+    def steady_state_latency(self, steady_state_time):  # pylint: disable=no-self-use
+        InfoEvent(message='Starting Steady State calculation for %ss' % steady_state_time).publish()
+        time.sleep(steady_state_time)
+        InfoEvent(message='Ended Steady State calculation. Took %ss' % steady_state_time).publish()
+
+    @latency_calculator_decorator
+    def post_upgrades_steady_state(self, steady_state_time):  # pylint: disable=no-self-use
```

Comment:
I think that this should be moved to the PerformanceRegressionUpgradeTest
because it is not used in PerformanceRegressionUpgradeTest and probably will never be used

Reply:
i'm doesn't fully agree that it will never be used, but also because of its similarity, if the `steady_state_calculation` is already in the parent class, this one has not reason to be in the child class, but IMHO should go along with that